rental.rb

```diff
@@ -1,6 +1,5 @@
 class Rental
-  attr_accessor :date
-  attr_reader :book, :person
+  attr_accessor :date, :person, :book
 
   def initialize(date, book, person)
     @date = date
```

Comment:
- [ ] As requested by the previous reviewer, please fix the rental error. When I try to create a rental it says that the push is not defined. Please initialize the method push.
### Error
![Screenshot 2022-11-04 155350](https://user-images.githubusercontent.com/81727988/199948321-d875afd3-87e7-4e79-87f2-7bfcdf7408c7.png)

Reply:
Hey @roshan-bajgain  👋 

This is working, please I suggest you add a person and a book first before testing the rentals since the rental depends on those two.

Thanks.

Reply:
@DuaneDave, I try twice but it was not working.